src/component/userProfile/userProfile.js

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import { BrowserRouter as Router, Switch, Route, Link} from "react-router-dom";
+import { Button, ButtonGroup, Container, ListGroup, Row, Col, Figure, Form } from 'react-bootstrap';
+import './userProfile.css';
+import Meeting from '../Meeting/meeting';
+import SearchPage from '../SearchPage/SearchPage';
+import TutorSubjectsAndTimes from '../tutorSubjectsAndTimes/TutorSubjectsAndTimes';
+
+const UserProfile = () => {
+
+    return <div className="container">
+
+    <Container>
+
+    <Row>
+        <h1><b>User Profile</b></h1>
+    </Row>
+
+<br></br>
```

Comment:
not a blocker, but just FYI this could just be one `</br>` i believe

Reply:
Got it, thanks.